plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -633,7 +726,7 @@ export const coreProfilerStateMachineDefinition = createMachine( {
 							actions: [
 								assign( {
 									userProfile: (
-										context,
+										_context,
```

Comment:
nitpick: Do we want to use _ for context? I understand it implies internal use, but I don't think we use this convention anywhere in our code base.

Reply:
It's actually because lint complains about unused variables when it's not used in the body of the function 🥲
Putting a '_' is an escape hatch in the lint rule for these situations where you need to declare the variable even when unused because we need to access the second parameter.

In xstate v5 it'll be changed to a destructured object so we won't need these _unusedvariables anymore